00_intro.txt

```diff
@@ -260,12 +259,10 @@ que la última está casi en lenguaje humano, inglés: `log`(registra) la
 operaciones como `sum` y `range`.)
 
 (((lenguaje de programación,poder de)))(((composición)))Un buen lenguaje de
-programación ayuda al programador mediante permitirle hablar acerca de las
-acciones que la computadora tiene que realizar en un nivel más alto.
-Ayuda a omitir detalles que no nos interesan, provee convenientes
+programación ayuda al programador a comunicar a la computadora, en un nivel más alto, las
+acciones que esta debe realizar. Esto ayuda a omitir detalles que son irrelevantes, proporciona convenientes
 bloques de  construcción (tales como `while` y `console.log`), y te permite
-definir tus propios bloques (como `sum` y `range`), y hace fácil componer esos
-bloques.
+definir tus propios bloques (como `sum` y `range`), y hace que sean más fáciles de componer dichos bloques.
```

Comment:
Creo que aquí los dos nos alejamos de la idea. Propongo algo como "definir tus propios bloques (como `sum` y `range`), y hace que sea más fácil integrarlos." El original dice: "...building blocks (such as `sum` and `range`), and makes those blocks easy to compose." **compose** se refiere a componer en el sentido de construir algo.